frontend/src/components/MainToolbar.vue

```diff
@@ -153,6 +154,11 @@ export default {
     ]),
     handleLogout () {
       this.$auth.signout()
+    },
+    showInfoDialog () {
+      this.$refs.infoDialog.showDialog()
```

Comment:
I thought we wanted to get rid of accesses to child components via $refs. Instead we could use v-model to control visibility.